packages/transaction-controller/src/TransactionController.ts

```diff
@@ -1132,6 +1133,7 @@ export class TransactionController extends BaseController<
       batchId?: Hex;
       deviceConfirmedOn?: WalletDevice;
       disableGasBuffer?: boolean;
+      fiatValue?: string;
```

Comment:
Do we definitely want a string? And would that be pre-formatted so `$1.23` or do we want Hex or a decimal string etc?